include/swift/SIL/SILType.h

```diff
@@ -763,9 +763,13 @@ class SILType {
   /// Returns true if this is the AnyObject SILType;
   bool isAnyObject() const { return getASTType()->isAnyObject(); }
 
-  /// Returns true if this type is a first class move only type or a move only
-  /// wrapped type.
-  bool isMoveOnly() const;
+  /// Returns true if this type is a noncopyable type. Otherwise, if the type
+  /// satisfies \c isMoveOnlyWrapped(), then it returns true iff \c orWrapped
+  /// is true. That is,
+  ///
+  /// orWrapped == false -->  isNoncopyable
+  /// orWrapped == true  -->  isNoncopyable || isMoveOnlyWrapped
+  bool isMoveOnly(bool orWrapped=true) const;
```

Comment:
At this occasion, can you rename this API to `isCopyable` (with the invert meaning)?

Reply:
Something that's wrapped in a SILMoveOnlyWrapper isn't necessarily noncopyable. It's just not implicitly copyable, so perhaps it should really imply that instead.